custom_components/kia_uvo/services.yaml

```diff
@@ -226,3 +226,21 @@ set_charge_limits:
             max: 100
             step: 10
             unit_of_measurement: '%'
+set_charging_current:
+  fields:
+    device_id:
+      required: false
+      selector:
+        device:
+          integration: kia_uvo
+    level:
+      required: true
+      selector:
+        select:
+          options:
+            - label: "100%"
+              value: 1
```

Comment:
Not sure why but the CI isn't happy with this: https://github.com/Hyundai-Kia-Connect/kia_uvo/actions/runs/9652140110/job/26621566236